assets/templates/datasetLandingPage/nomis.tmpl

```diff
@@ -0,0 +1,245 @@
+<div class="page-intro">
```

Comment:
background--gallery is added to this class in other templates. As most of it is similar to filterable.tmpl, is it possible to just create a common template and call it where necessary? It feels as we are duplicating a lot

Reply:
It was decided (convo with James we are taking a baby step towards going in this direction https://deploy-preview-3--ons-dp-prototypes.netlify.app/prototypes/nomis-dataset-page/#nomis but not implementing that design entirely.

Reply:
I am worried that we might miss this class (if it is important) as it is a very small one. And as a whole the template looks similar so it might not be picked up that easily. Especially like me. Is it ok to add it?